proto/rill/admin/v1/api.proto

```diff
@@ -814,4 +895,15 @@ message UserInvite {
 message WhitelistedDomain {
   string domain = 1;
   string role = 2;
+}
+
+message DashboardBookmark {
```

Comment:
For consistency, should just be `Bookmark` (since the APIs are all `CreateBookmark`, etc.)